frontend/src/app/pages/landing-page/landing-page/landing-page.component.html

```diff
@@ -9,12 +9,11 @@ <h1 class="display-3 m-3 text-center">Digitale FüSim MANV</h1>
         <div class="input-group">
             <input
                 [(ngModel)]="exerciseId"
+                (ngModelChange)="joinInputChanged()"
```

Comment:
I believe this would break if you switch the `[(ngModel)]` and `(ngModelChange)`.

Reply:
Background:

```
[(ngModel)]="exerciseId"
``` 
is the shorthand syntax for 
```
[ngModel]="exerciseId"
(ngModelChange)="exerciseId = $event"
```

Reply:
Thank you for the warning. Since I'm using the paste event instead of the change event now, this is resolved.